Maddori.Apple/Maddori.Apple/Screen/AddFeedbackContent/AddFeedbackContentViewController.swift

```diff
@@ -98,72 +144,114 @@ final class AddFeedbackContentViewController: BaseViewController {
             $0.top.equalTo(feedbackContentLabel.snp.bottom).offset(SizeLiteral.labelComponentPadding)
-            $0.leading.equalTo(addFeedbackContentView.snp.leading).inset(SizeLiteral.leadingTrailingPadding)
-            $0.trailing.equalTo(addFeedbackContentView.snp.trailing).inset(SizeLiteral.leadingTrailingPadding)
-            $0.height.greaterThanOrEqualTo(150)
+            $0.leading.trailing.equalToSuperview().inset(SizeLiteral.leadingTrailingPadding)
+            $0.height.equalTo(150)
+        }
+        
+        addFeedbackContentView.addSubview(feedbackStartSwitch)
+        feedbackStartSwitch.snp.makeConstraints {
+            $0.top.equalTo(feedbackContentTextView.snp.bottom).offset(SizeLiteral.componentIntervalPadding)
+            $0.trailing.equalToSuperview().inset(SizeLiteral.leadingTrailingPadding)
+            $0.width.equalTo(51)
+            $0.height.equalTo(31)
         }
         
-        addFeedbackContentView.addSubview(feedbackDoneButton)
+        addFeedbackContentView.addSubview(feedbackStartLabel)
+        feedbackStartLabel.snp.makeConstraints {
+            $0.centerY.equalTo(feedbackStartSwitch.snp.centerY)
+            $0.leading.equalToSuperview().inset(SizeLiteral.leadingTrailingPadding)
+        }
+        
+        addFeedbackContentView.addSubview(feedbackStartTextViewLabel)
+        feedbackStartTextViewLabel.snp.makeConstraints {
+            $0.top.equalTo(feedbackStartSwitch.snp.bottom).offset(SizeLiteral.componentIntervalPadding)
+            $0.leading.equalToSuperview().inset(SizeLiteral.leadingTrailingPadding)
+        }
+        
+        addFeedbackContentView.addSubview(feedbackStartTextView)
+        feedbackStartTextView.snp.makeConstraints {
+            $0.top.equalTo(feedbackStartTextViewLabel.snp.bottom).offset(SizeLiteral.labelComponentPadding)
+            $0.leading.trailing.equalToSuperview().inset(SizeLiteral.leadingTrailingPadding)
+            $0.height.equalTo(150)
+        }
+        
+        view.addSubview(feedbackDoneButtonView)
+        feedbackDoneButtonView.snp.makeConstraints {
+            $0.bottom.equalToSuperview()
+            $0.leading.trailing.equalToSuperview()
+            $0.height.equalTo(134)
+        }
+        
+        feedbackDoneButtonView.addSubview(feedbackDoneButton)
         feedbackDoneButton.snp.makeConstraints {
-            $0.bottom.equalTo(view.safeAreaLayoutGuide).inset(2)
-            $0.centerX.equalTo(addFeedbackContentView.snp.centerX)
-            $0.width.equalTo(addFeedbackContentView.snp.width).inset(SizeLiteral.leadingTrailingPadding)
+            $0.bottom.equalTo(feedbackDoneButtonView.snp.bottom).inset(36)
+            $0.leading.trailing.equalToSuperview().inset(SizeLiteral.leadingTrailingPadding)
+        }
+        
+        feedbackDoneButtonView.addSubview(feedbackSendTimeLabel)
+        feedbackSendTimeLabel.snp.makeConstraints {
+            $0.bottom.equalTo(feedbackDoneButton.snp.top).inset(-11)
```

Comment:
p5;

```suggestion
            $0.bottom.equalTo(feedbackDoneButton.snp.top).offset(-11)
```
요 부분은 offset으로 변경 가능할까요 ?? 사소하지만 저번에 슬랙에서 @seongmin221 께서 말해준 것 처럼 inset은 superView로부터 위치를 잡을때 사용하고, offset은 component간의 위치를 잡을때 사용한다는 얘기가 있어서 통일하려 함입니다 !

Reply:
[3a4baa3](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/47/commits/3a4baa3944e6ce9fc7586a4ba21203ee03f074af)
수정 완료 ! ⭐️ 감사합니다 :)